src/modules/75-cf/pages/feature-flags/components/FeatureFlagsListing.tsx

```diff
@@ -102,6 +117,33 @@ const FeatureFlagsListing: FC<FeatureFlagsListingProps> = ({
 
   const columns: Column<Feature>[] = useMemo(
     () => [
+      {
+        id: 'staleCheckbox',
+        Header: (
+          <Checkbox
+            aria-label="selectAllFlags"
+            onClick={() => {
+              //TOD: Handle Select all
+            }}
+          />
+        ),
+        accessor: row => row.kind,
+        width: '22%',
+        //TODO: Remove this once var is used
+        // eslint-disable-next-line no-empty-pattern
+        Cell: ({}: CellProps<Feature>) => {
+          return (
+            <Checkbox
+              onClick={event => {
+                //TODO: Handle selection of row
+                event.stopPropagation()
+                onCheckboxSelect(event)
+              }}
+            />
```

Comment:
You can kill the click and prevent the checkbox being stretched (not visually, but in the DOM) by putting in a wrapping component.
```suggestion
            <div onClick={e => e.stopPropagation()}>
              <Checkbox onClick={onCheckboxSelect} />
            </div>
```

Reply:
Nice thank you